### PR TITLE
doc(readme): add branch rules

### PR DESCRIPTION
**Pull request by kwiss**

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -6,10 +6,8 @@
       "message": "chore(release): version %v"
     }
   },
-  "packages": [
-    "optional/*",
-    "packages/*"
-  ],
+  "packages": ["optional/*", "packages/*"],
+  "useWorkspaces": true,
```
Comment (douglasduteil) [inline lerna.json]: As we define different packages for `yarn` and `lerna` we are not `use[ing]Workspaces` the workspace key in the package.json

File: package.json [hunk 2]
```diff
@@ -1,5 +1,7 @@
 {
+  "name": "@socialgouv/empjm",
```
Comment (douglasduteil) [inline package.json]: Necessary for ?

File: package.json [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
   "private": true,
+  "description": "Trouver rapidement le bon professionnel pour les majeurs à protéger",
```
Comment (douglasduteil) [inline package.json]: Necessary for ?

File: README.md [hunk 4]
```diff
@@ -71,3 +70,23 @@ $ CONVENTIONAL_GITHUB_RELEASER_TOKEN==************ npx conventional-github-relea
 #
 # You might want to add a Gif to your release to make it groovy ;)
 ```
+
+### Branch name
+
+You need to name your branch with the commit lint convention
```
Comment (douglasduteil) [inline README.md]: may want to link to conventionalcommits.org and https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular

Comment (kwiss) [reply README.md]: done

**Pull request merged**